src/lockfiles.nim

```diff
@@ -250,23 +227,24 @@ proc convertNimbleLock*(c: var AtlasContext; nimblePath: string): LockFile =
 
   if jsonTree.getOrDefault("version") == nil or
       "packages" notin jsonTree:
-    error c, toRepo(nimblePath), "invalid nimble lockfile"
+    error c, nimblePath, "invalid nimble lockfile"
     return
 
   result = newLockFile()
   for (name, info) in jsonTree["packages"].pairs:
     if name == "nim":
       result.nimVersion = info["version"].getStr
-      continue
-    # lookup package using url
-    let pkg = c.resolvePackage(info["url"].getStr)
-    info c, toRepo(name), " imported "
-    let dir = c.depsDir / pkg.repo.string
-    result.items[name] = LockFileEntry(
-      dir: dir.relativePath(c.projectDir),
-      url: $pkg.url,
-      commit: info["vcsRevision"].getStr,
-    )
+    else:
+      # lookup package using url
+      let pkgurl = info["url"].getStr
+      info c, name, " imported "
+      let u = createUrl(pkgurl, c.overrides)
+      let dir = c.depsDir / u.projectName
```

Comment:
Pretty sure this is wrong and would break the `repo.user.url` setup and possibly other edge cases.

Reply:
The `repo.user.url` feature has not been replicated.